pkg/util/k8s/k8s.go

```diff
@@ -2275,3 +2277,33 @@ func AddManagedByOperatorLabel(om metav1.ObjectMeta) metav1.ObjectMeta {
 	om.Labels[constants.OperatorLabelManagedByKey] = constants.OperatorLabelManagedByValue
 	return om
 }
+
+func GetDefaultKubeControllerManagerImage(k8sVersion *version.Version) string {
+	if k8sVersion == nil {
```

Comment:
nit: Can we add a quick UT for this condition here and in `GetDefaultKubeSchedulerImage`?